enlighten/BusinessObjects.py

```diff
@@ -663,13 +663,7 @@ def create_rest(self):
             spinbox                     = sfu.spinBox_integration_time_ms)
 
         self.header("instantiating GainDBFeature")
-        ctl.gain_db_feature = GainDBFeature(ctl = ctl,
-            bt_dn                       = sfu.pushButton_gain_dn,
-            bt_up                       = sfu.pushButton_gain_up,
-            label                       = sfu.label_gainWidget_title,
-            multispec                   = ctl.multispec,
-            slider                      = sfu.slider_gain,
-            spinbox                     = sfu.doubleSpinBox_gain)
+        ctl.gain_db_feature = GainDBFeature(ctl = ctl)
```

Comment:
I basically disagree with the premise that the application is improved by making everything global (every business object has access to basically everything in the application, including accessing every GUI widget through a flat namespace), but I'm willing to accept it.  You can quote all the PEPs you want — I just place a higher priority on encapsulation.  Anyway, I'm not fighting it, and I do believe that the primary maintainer of a codebase has the right to move things around to suit their preferences (I did).  I will try to learn and follow your style so we're contributing to a consistent design.

Reply:
I think of it as a lesser of two evils. I lean towards 'the ctl' way because it makes it easier to track references via grep or static analysis, and semantic changes can be performed faster.